data.md

```diff
@@ -318,23 +301,9 @@ This stack also serves as a cons-list, so we provide some standard cons-list man
     rule WS [ START .. WIDTH ] => #take(chop(WIDTH), #drop(chop(START), WS))
 ```
 
--   `WS [ N ]` accesses element $N$ of $WS$.
```

Comment:
I'm surprised the `WS [ N ]` lookup was never used.

Reply:
Ah, EVM had a stack but in IELE the only indexing we do is reading slices from memory locations, and we always want a range in that case.